xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/dataeditors.css

```diff
@@ -323,20 +323,21 @@ div#xwikiobjects input[type="radio"], div#xwikiclassproperties input[type="radio
   text-decoration: underline;
   background-color: $theme.highlightColor;
 }
-
+#add_xobject .buttonwrapper input , #add_xproperty .buttonwrapper input {
+    vertical-align: top;
+    margin: 0px;
```

Comment:
Why not use 2 spaces to indent, like the rest of the CSS code in this file? :) Also, when the value is ``0`` the unit doesn't matter so it can be omitted.

Reply:
The 'Add' button is given `margin: 2px 0px` on the top of this CSS file. So I prefer to make it zero rather than changing from the top. I thought to omit the `margin: 2px` may affect other components :). But let me make this cleaner.